src/styles/blocks/cards.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  width: 166px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.card__image {
+  display: flex;
+  margin: auto;
+  margin-bottom: 40px;
+}
+
+.card__title {
+  font-family: "Roboto", sans-serif;
+  font-weight: normal;
+  color: var(--fontMainColor);
+  margin-bottom: 4px;
+
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__productCode {
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  font-weight: lighter;
+  font-family: "Roboto-Regular", sans-serif;
```

Comment:
Your code looks great! 

Maybe it will make sense just to put 'font-family' to the container (.card) to not duplicate this value/